### PR TITLE
Add entities to DAL

### PR DESCRIPTION
**Pull request by hinkevich**

### CODE REVIEW

File: GymApp/GYM.DAL/Entities/Couch.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace GYM.API.Models
```
Comment (stas1408123) [inline GymApp/GYM.DAL/Entities/Couch.cs]: Why namespace different?

File: GymApp/GYM.DAL/Entities/Couch.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+{
+    public class Couch
```
Comment (stas1408123) [inline GymApp/GYM.DAL/Entities/Couch.cs]: Please read task description

File: GymApp/GYM.DAL/Entities/Couch.cs [hunk 3]
```diff
@@ -0,0 +1,22 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace GYM.API.Models
+{
```
Comment (Genresal) [inline GymApp/GYM.DAL/Entities/Couch.cs]: FIx namespaces, please.

File: GymApp/GYM.DAL/Entities/Couch.cs [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    public class Couch
+    {
+        public int Id { get; set; }
+        [Required]
+        [StringLength(150, MinimumLength = 1)]
+        public string FirstName { get; set; } = null!;
+        [Required]
```
Comment (Genresal) [inline GymApp/GYM.DAL/Entities/Couch.cs]: Not sure that validation attributes are necessary for entities.

**Review by Genresal — APPROVED** [review]

**Review by stas1408123 — APPROVED** [review]

**Review by SasukeVapes — APPROVED** [review]

**Pull request merged**